pkg/util/trivy/cache.go

```diff
@@ -198,19 +166,22 @@ func trivyCachePut[T cachedObject](cache *ScannerCache, id string, info T) error
 func trivyCacheGet[T cachedObject](cache *ScannerCache, id string) (T, error) {
 	rawValue, err := cache.Cache.Get(id)
 	var empty T
-
 	if err != nil {
 		return empty, fmt.Errorf("error getting object with ID %q from cache: %w", id, err)
 	}
-
 	var res T
 	if err := json.Unmarshal(rawValue, &res); err != nil {
 		return empty, fmt.Errorf("JSON unmarshal error: %w", err)
 	}
-
 	return res, nil
 }
 
+// ScannerCache wraps a generic Cache and implements trivy.Cache.
```

Comment:
Moved closer to where its methods are implemented. The file is very large and it's easier to have this definition close to its method